### PR TITLE
Initial Commit

### PR DESCRIPTION
**Pull request by SquirtleSquadLeader**
1) Corrected Filter_Bandwidth register bit location
2) Add register definitions for Interrupt Enables/Status
3) Add getter/setter for interrupts
4) Add External Sync Set feature

### CODE REVIEW

**Review by tekktrik — CHANGES_REQUESTED** [review]
I'll test this on my sensor, but in the meantime, I would reword the docstrings for all of the `@property` functions with at least a one liner that will help others understand what they're accessing.  This is important because it will show up in the online documentation.  It looks like you have it for some, so just the ones that don't already!

**Comment by SquirtleSquadLeader** [comment]
> I'll test this on my sensor, but in the meantime, I would reword the docstrings for all of the `@property` functions with at least a one liner that will help others understand what they're accessing. This is important because it will show up in the online documentation. It looks like you have it for some, so just the ones that don't already!

@tekktrik Can you confirm the property.setter's require them as well???

**Comment by tekktrik** [comment]
The setters do not get propagated to the online documentation, so technically only the getter ones matter.  They should therefore describe the property that is get and/or set.

**Comment by SquirtleSquadLeader** [comment]
> The setters do not get propagated to the online documentation, so technically only the getter ones matter. They should therefore describe the property that is get and/or set.

@tekktrik Okay, I am going to get up in the AM and take care of any missing doc-strings.  Thank you for the help!

**Comment by tekktrik** [comment]
You don't have to close this as completed if you still want someone to take a look, just so you know!
